core/src/main/java/io/fabric8/maven/core/service/kubernetes/JibBuildConfiguration.java

```diff
@@ -35,6 +35,8 @@
 
     private String outputDir;
 
+    private String tarImageName;
```

Comment:
Not required. We can figure out the tarImageName from the information we have in the buildImage Utility function.